__tests__/Conversion.ts

```diff
@@ -1,4 +1,11 @@
-import { fromJS, is, List, Map, OrderedMap, Record } from 'immutable';
+import {
+  fromJS,
+  is,
+  List,
+  Map,
+  OrderedMap,
+  Record as ImmutableRecord,
```

Comment:
Instead of aliasing the `Record` name here, can you just make an explicit type in the unit test? (Does that value even need to be explicitly typed?)